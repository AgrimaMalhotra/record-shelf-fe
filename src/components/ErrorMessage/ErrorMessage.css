.error-container {
  padding-left: 30px;
}