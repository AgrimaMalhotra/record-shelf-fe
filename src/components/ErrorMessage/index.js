import React from 'react';
import propTypes from 'prop-types';
import './ErrorMessage.css';
const ErrorMessage = ({ errorMessage }) => {
  return (
    <div className="error-container">
      <h3>Something Went Wrong!</h3>
      {errorMessage && <h5>{`Error code:${errorMessage}`}</h5>}
    </div>
  );
};

ErrorMessage.propTypes = {
  errorMessage: propTypes.oneOfType([propTypes.string, propTypes.number]),
};
export default ErrorMessage;
