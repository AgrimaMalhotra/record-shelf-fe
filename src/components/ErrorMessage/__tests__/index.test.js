import { render } from '@testing-library/react';
import * as React from 'react';
import ErrorMessage from '..';

describe('Error Page', () => {
  it('should render correctly ', () => {
    const { container } = render(<ErrorMessage />);
    expect(container).toMatchSnapshot();
  });
});
