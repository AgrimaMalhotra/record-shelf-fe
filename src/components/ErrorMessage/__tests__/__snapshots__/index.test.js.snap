// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page should render correctly  1`] = `
<div>
  <div
    class="error-container"
  >
    <h3>
      Something Went Wrong!
    </h3>
  </div>
</div>
`;
